tests/testutils/iputils.go

```diff
@@ -83,8 +83,32 @@ func GetIPsv6(iface string) ([]string, error) {
 }
 
 func GetGW(iface string, family int) (string, error) {
-	return "", fmt.Errorf("Not implemented")
+	var ret string
+	list, err := net.InterfaceByName(iface)
+	if err != nil {
+		return "", err
+	}
+
+	addrs, err := list.Addrs()
+	if err != nil {
+		return "", err
+	}
+	for _, addr := range addrs {
```

Comment:
The previous version of the patch got the gateway by checking the routes. But this version just returns the IP on the interface. I am not sure what "GetGW()" means then.